TestApp/update-npm-packages.sh

```diff
@@ -13,32 +42,49 @@ npm pack ../appcenter-link-scripts
 echo "Installing appcenter* packages..."
 npm install appcenter*.tgz
 
-echo "workaround for macs on arm64"
-npm install react-native-fs --save --legacy-peer-deps
-
 echo "Cleanup appcenter*.tgz"
 rm appcenter*.tgz
 
 echo "Installing other packages..."
 npm install
 
-# Issue: https://github.com/react-native-community/cli/issues/1403
-# Fixed in RN 0.65: https://github.com/facebook/react-native/commit/8721ee0a6b10e5bc8a5a95809aaa7b25dd5a6043
-echo "Applying patch to react-native package that fixes build on Xcode 12.5..."
-npx patch-package
+if [ "$is_ios" == "true" ]; then
+    if [[ "$OSTYPE" == "darwin"* && "$(uname -m)" == "arm64" ]]; then
+        echo "Workaround for macs on arm64"
+        npm install react-native-fs --save --legacy-peer-deps
```

Comment:
what is this `react-native-fs` used for and how it fixes the build on arm64?

Reply:
This is not a new code, this code was implemented here https://github.com/microsoft/appcenter-sdk-react-native/pull/921